app/controllers/tag_articles_controller.rb

```diff
@@ -0,0 +1,65 @@
+class TagArticlesController < ApplicationController
+  before_action :set_cache_control_headers, only: :index
+
+  SIGNED_OUT_RECORD_COUNT = 60
+
+  rescue_from ArgumentError, with: :bad_request
```

Comment:
what's the scenario for this?

Reply:
I didn't check this, thanks for the reminder! This is taken directly from the original controller which was introduced In [PR 6248](https://github.com/forem/forem/blob/72d0251bc5184afffc5475615ec1272cbeb506a7/app/controllers/stories_controller.rb#L21)
https://github.com/forem/forem/blob/72d0251bc5184afffc5475615ec1272cbeb506a7/app/controllers/stories_controller.rb#L21

This will handle the error produced by the invalid character in the URL.  In that PR this occurred with the slug of `Article`.
If this scenario won't happen in the name of `Tag`, we can remove it.